apisix/http/route.lua

```diff
@@ -21,7 +21,7 @@ local service_fetch = require("apisix.http.service").get
 local core = require("apisix.core")
 local expr = require("resty.expr.v1")
 local plugin_checker = require("apisix.plugin").plugin_checker
-local ai = require("apisix.core.ai")
+local internal_event = require("apisix.internal_event")
```

Comment:
I think `event` is a good name, we can remove `internal_`

Reply:
update